node_modules/oae-core/activity/js/activity.js

```diff
@@ -122,8 +122,9 @@ define(['jquery', 'underscore', 'oae.core', 'activityadapter'], function($, _, o
             infinityScroll = $('.oae-list', $rootel).infiniteScroll(url, {
                 'limit': 10
             }, '#activity-items-template', {
-                'postProcessor': processActivities,
-                'emptyListProcessor': handleEmptyResultList
+                'ariaLive': 'off',  // Notifications will alert screen readers
```

Comment:
Also, FWIW, if we do want to clue screen readers on some events in the activity stream (e.g. those that aren't notifications), I think `aria-live="polite"` is probably better than `aria-live="assertive"`. As @simong notes, we've already made the determination that those aren't critical events because we've omitted them from notifications.